internal/git/rebase.go

```diff
@@ -49,6 +52,15 @@ func (r *Repo) Rebase(opts RebaseOpts) (*Output, error) {
 			Args: []string{"rebase", "--skip"},
 		})
 	}
+
+	if opts.Interactive {
+		args = append(args, "-i")
+		args = append(args, "master")
```

Comment:
I don't think you need this. You should just add "-i" and run in interactive. Other args are added in other parts of the code.

Reply:
got it. I've pushed the changes, but I still have issue with the text editor. Seems like the fancy text that show up during non interactive is still showing up during interactive session. I've sent you the image on discord

Reply:
Hm. I think you would need to change this to use https://pkg.go.dev/github.com/charmbracelet/bubbletea#Exec or https://pkg.go.dev/github.com/charmbracelet/bubbletea#ExecProcess. Bubbletea tries to control the tty, so we should stop it while executing another process.